silk-workspace/src/main/scala/org/silkframework/workspace/activity/workflow/LocalWorkflowExecutor.scala

```diff
@@ -49,7 +52,10 @@ case class LocalWorkflowExecutor(workflowTask: ProjectTask[Workflow],
                   (implicit userContext: UserContext): Unit = {
     cancelled = false
     try {
+      val registry = MeterRegistryProvider.meterRegistry
+      val stopwatch: Timer.Sample = Timer.start()
       runWorkflow(context, updateUserContext(userContext))
+      stopwatch.stop(registry.timer(s"$prefix.workflow.execution", "workflow", workflowTask.id.toString))
```

Comment:
This would not include the execution time of failed workflows. That's probably fine, but we should ask for feedback if this is expected.